### PR TITLE
Alot of the changes made to the separate components..

### PR DESCRIPTION
**Pull request by JavachipMom**

### CODE REVIEW

File: kids-fly/package.json [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
     "@testing-library/user-event": "^7.1.2",
     "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
     "formik": "^2.1.4",
+    "moment": "^2.24.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.0",
     "react-axios": "^2.0.3",
     "react-dom": "^16.12.0",
-    "react-formik-ui": "^4.0.8",
```
Comment (hebergonza719) [inline kids-fly/package.json]: I need to have this

File: kids-fly/src/App.js [hunk 2]
```diff
@@ -12,7 +11,7 @@ import ParentRegister from "./components/ParentRegister";
 import PrivateRoute from './components/PrivateRoute';
 
 function App() {
-  const[newTrip, setNewTrip] = useState([])
+  const [newTrip, setNewTrip] = useState([])
```
Comment (hebergonza719) [inline kids-fly/src/App.js]: This doesn't matter

File: kids-fly/src/index.js [hunk 3]
```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import './index.css';
-import App from './App';
```
Comment (hebergonza719) [inline kids-fly/src/index.js]: I do need this import

File: kids-fly/src/index.js [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-import * as serviceWorker from './serviceWorker';
-import { BrowserRouter as Router} from "react-router-dom";
+import LandingPage from './components/LandingPage';
+import Header from './components/Header';
 
-ReactDOM.render(
-  <Router>
-    <App />
-  </Router>, 
-  document.getElementById('root'));
```
Comment (hebergonza719) [inline kids-fly/src/index.js]: lines 8 thru 12 do the same thing lines 17 thru 23 do

Comment (JavachipMom) [reply kids-fly/src/index.js]: Got it! I did resolve some of the merge conflicts..

**Pull request merged**